apps/client/src/components/ConclusionOutcome.tsx

```diff
@@ -0,0 +1,147 @@
+import { Button, Heading, ListItem, Paragraph } from "@datapunt/asc-ui";
```

Comment:
Can you move this file to components/Conclusion/ConclusionOutcome.tsx ? So we can add all conclusion related files to this.